eslint.config.cjs

```diff
@@ -1,51 +1,51 @@
-const { FlatCompat } = require('@eslint/eslintrc');
-const js = require('@eslint/js');
-const nextPlugin = require('@next/eslint-plugin-next');
-const tsPlugin = require('@typescript-eslint/eslint-plugin');
-const tsParser = require('@typescript-eslint/parser');
+const { FlatCompat } = require("@eslint/eslintrc");
+const js = require("@eslint/js");
+const nextPlugin = require("@next/eslint-plugin-next");
+const tsPlugin = require("@typescript-eslint/eslint-plugin");
+const tsParser = require("@typescript-eslint/parser");
 
 const compat = new FlatCompat();
 
 module.exports = [
   js.configs.recommended,
-  ...compat.extends('next/core-web-vitals', 'next/typescript'),
+  ...compat.extends("next/core-web-vitals", "next/typescript", "prettier"),
```

Comment:
The only change in this file is the addition of `"prettier"`. The rest are due to automated formatting.